pyproject.toml

```diff
@@ -0,0 +1,78 @@
+[project]
+name = "odoo-test-helper"
+authors = [
+    {name = "ACSONE", email="info@acsone.eu"},
+    {name = "Akretion", email="info@akretion.com"},
+    {name = "Camptocamp", email="info@camptocamp.com"},
+]
+description = "Our Odoo project tools"
+readme = "README.rst"
+keywords = ["odoo", "project"]
+license = {file = "LICENSE"}
+classifiers = [
+    "License :: OSI Approved :: GNU Lesser General Public License v3 or later (LGPLv3+)",
+    "Programming Language :: Python :: 3",
+    "Framework :: Odoo",
+    "Development Status :: 2 - Pre-Alpha",
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+]
+dependencies = [
+]
+requires-python = ">=3.9"
```

Comment:
@simahawk This will make module not installable for Odoo versions < v16. See logs

Reply:
fixed